appengine_flexible/websockets/main.go

```diff
@@ -70,9 +68,4 @@ func socketHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-// healthCheckHandler is used by App Engine Flex to check instance health.
-func healthCheckHandler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprint(w, "ok")
-}
-
```

Comment:
Healthchecks are noted in the [docs for websockets](https://cloud.google.com/appengine/docs/flexible/go/using-websockets-and-session-affinity?hl=en) as part of how the platform relocates sessions to preserve connections using session affinity. Migrating to split healthchecks seems valuable.

Reply:
That doc mentions "If the target instance fails health checks, App Engine moves the session to a healthy instance." That seems very reasonable.

https://cloud.google.com/appengine/docs/flexible/custom-runtimes/configuring-your-app-with-app-yaml#updated_health_checks mentions health checks are on by default and only need to be customized if you want to. So, I'm not sure it's valuable to add health checks here. Maybe we should update the websocket doc to link to the health check docs?

Reply:
Agreed, let's remove this and cross-link the docs. I'll send a CL to the docs now.